src/Uno.UI.Wasm/WasmScripts/Uno.UI.d.ts

```diff
@@ -251,6 +251,19 @@ declare module Uno.UI.Interop {
         main_class: Interop.IMonoClassHandle;
     }
 }
+declare module Uno.UI.Interop {
+    interface IWebAssemblyModule {
```

Comment:
The term `Module` is already used in TS/JS for something else :-|

Reply:
True, but emscripten uses this already... I'm not sure I can change it.